src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -714,15 +713,9 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
-  const isJazzbar = templateID === VenueTemplate.jazzbar;
-
-  const jukeboxContainerClasses = classNames("toggle-room DetailsForm", {
```

Comment:
maybe would be better to add the class here?
I'd like to remove as little as possible, not to overcomplicate the deployment process resolving merge conflicts.
```js
const jukeboxContainerClasses = classNames("toggle-room DetailsForm DetailsForm--hidden", {
```

will that work?

Reply:
sure, fixed e2041a0